test/root/scripts/test_class.ash

```diff
@@ -2,4 +2,5 @@ boolean ok;
 ok = ($class[Avatar of Sneaky Pete] == $class[Avatar of Sneaky Pete]);
 print("comparison " + to_string(ok));
 ok = ($class[Gelatinous Noob].primestat == $stat[Moxie]);
-print("primestat " + to_string(ok));
\ No newline at end of file
+print("primestat " + to_string(ok));
+print($class[Ed] == $class[Ed the Undying]);
```

Comment:
nit: missing newline (already existing, so I don't care that much if you fix it)